src/test/java/spoon/test/field/FieldTest.java

```diff
@@ -216,7 +216,7 @@ public void bugAfterRefactoringImports() {
 				"import java.io.File;\n" +
 				"class A {\n" +
 				"    public static final String separator = File.separator;\n" +
-				"}", klass.toStringWithImports());
+				"}\n", klass.toStringWithImports());
```

Comment:
Are you sure a unix newline is what we want here?

Reply:
The `@ExtendWith(LineSeparatorExtension.class)` helps us here. I just wanted to keep the same style in the testcase. We could convert this to a textblock?